pkgs/tools/X11/oblogout/oblogout-0.2-fixes.patch

```diff
@@ -0,0 +1,69 @@
+diff -urN oblogout.orig/data/oblogout oblogout/data/oblogout
+--- oblogout.orig/data/oblogout	2009-04-11 09:14:27.000000000 -0300
++++ oblogout/data/oblogout	2016-10-20 07:45:46.214715811 -0200
+@@ -79,8 +79,11 @@
+                 
+     # Check config in local path, if it exists pass it on     
+     if not os.path.exists(config):
+-        logger.error("Invalid config file: %s" % config)
+-        return 1
++        if not local_mode and os.path.exists(sys.prefix + config):
++            config = sys.prefix + config
++        else:
++            logger.error("Invalid config file: %s" % config)
++            return 1
+ 
+     # If debug mode is enabled, output debug messages
+     if verbose:
+diff -urN oblogout.orig/data/oblogout.conf oblogout/data/oblogout.conf
+--- oblogout.orig/data/oblogout.conf	2009-04-11 09:14:27.000000000 -0300
```

Comment:
this patch does change the default configuration, which is not used anyway, because of the module above.